mods/ts/rules/infantry.yaml

```diff
@@ -268,15 +257,10 @@ JUMPJET:
 
 CHAMSPY:
 	Inherits: ^Infantry
-	Buildable:
-		Queue: Infantry
-		BuildPaletteOrder: 60
-		Prerequisites: ~nahand, natech
 	Valued:
 		Cost: 700
 	DisguiseToolTip:
 		Name: Chameleon Spy
-		Description: Infiltrates enemy structures to gather \nintelligence. Exact effect depends on the \nbuilding infiltrated.\n  Strong vs Nothing\n  Weak vs Everything\n  Special Ability: Disguised
```

Comment:
They aren't used (since they aren't buildable), and some of them are completely wrong.